app/Services/ReportbackService.php

```diff
@@ -14,14 +14,14 @@ class ReportbackService
     protected $reportbackRepository;
 
     /*
-     * Instance of \Rogue\Services\Phoenix\Phoenix
+     * Instance of \Rogue\Jobs\SendReportbackToPhoenix
      */
-    protected $phoenix;
+    protected $sendReportbackToPhoenix;
 
-    public function __construct(ReportbackRepository $reportbackRepository, Phoenix $phoenix)
+    public function __construct(ReportbackRepository $reportbackRepository, SendReportbackToPhoenix $sendReportbackToPhoenix)
     {
         $this->reportbackRepository = $reportbackRepository;
-        $this->phoenix = $phoenix;
+        $this->sendReportbackToPhoenix = $sendReportbackToPhoenix;
```

Comment:
ah shoot yes, I forgot to delete that, thanks @DFurnes ! I'll make the changes and submit a new PR!